build.sbt

```diff
@@ -434,6 +438,7 @@ lazy val `play-json` = (project in file("play-json"))
     name := "lagom-scaladsl-play-json",
     Dependencies.`play-json`,
     mimaBinaryIssueFilters ++= Seq(
+      // see https://github.com/lagom/lagom/pull/189
```

Comment:
Yeah, #189 added the method originally but the breaking change was introduced in https://github.com/lagom/lagom/pull/1071/files#diff-3dbdc8eedd6ae885efe6d4f880930a4bR85

Reply:
whoops, I will update that